src/Microsoft.AspNet.OData.Shared/Common/SRResources.resx

```diff
@@ -548,7 +548,7 @@
     <value>The property 'EdmObject' of {0} cannot be null.</value>
   </data>
   <data name="BatchRequestInvalidMediaType" xml:space="preserve">
-    <value>The batch request must have '{0}' as the media type.</value>
+    <value>The batch request must have '{0}' or '{1}' as the media type.</value>
```

Comment:
Should we just hardcode the types?

Reply:
I think it should be good to keep the message and data separate.

---
In reply to: [198253383](https://github.com/OData/WebApi/pull/1504#discussion_r198253383) [](ancestors = 198253383)